source/_integrations/onvif.markdown

```diff
@@ -7,49 +7,19 @@ ha_release: 0.47
 ha_domain: onvif
 ---
 
-The `onvif` camera platform allows you to use an [ONVIF](https://www.onvif.org/) camera in Home Assistant. This requires the [`ffmpeg` component](/integrations/ffmpeg/) to be already configured.
+The `onvif` camera platform allows you to use an [ONVIF](https://www.onvif.org/) Profile S conformant device in Home Assistant. This requires the [`ffmpeg` component](/integrations/ffmpeg/) to be already configured.
 
 ## Configuration
 
-To enable your ONVIF camera in your installation, add the following to your `configuration.yaml` file:
+Home Assistant offers UniFi integration through **Configuration** -> **Integrations** -> **ONVIF**. Follow the instructions to get it set up.
```

Comment:
![image](https://user-images.githubusercontent.com/195327/80650255-f1c03880-8a73-11ea-93bd-58c66e9891dd.png)

Sorry! I could not resist myself!


```suggestion
Home Assistant offers ONVIF integration through **Configuration** -> **Integrations** -> **ONVIF**. Follow the instructions to get it set up.
```